browser/net/brave_block_safebrowsing_urls.cc

```diff
@@ -16,15 +17,15 @@ namespace brave {
 const char kDummyUrl[] = "https://no-thanks.invalid";
 
 bool IsSafeBrowsingReportingURL(const GURL& gurl) {
-  static const std::vector<URLPattern> allowed_patterns({
+  static const base::NoDestructor<std::vector<URLPattern>> allowed_patterns({
```

Comment:
are these changes required for things to work or it's just to fix the codestyle? If it's the latter, I'd prefer to have this as a separate PR.

Reply:
Unfortunately, this was needed as CI failed for 'declaration requires an exit-time destructor'

Reply:
I see. That's understandable then.